docs/classical_control.ipynb

```diff
@@ -0,0 +1,311 @@
+{
```

Comment:
<p>"Will generally avoid" is worrying because it implies that this might occur in some situations and make the user's circuit invalid. Is this guaranteed for all transformers provided by cirq? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/5403/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
<p>That's right, all transformers should ensure that CCOs don't move across their corresponding measurements (and vice versa) -- if we find a case where this is not true then it's a bug and should be fixed.</p>